frontend/internal-packages/configs/biome.jsonc

```diff
@@ -1,38 +1,82 @@
 {
   "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
-  "organizeImports": {
-    "enabled": true
+  "assist": { 
+    // TODO: Re-enable after handling large diff impact across codebase
+    "enabled": false
```

Comment:
The behavior of organizeImports has changed significantly and now produces a large number of diffs. 
ref: https://biomejs.dev/assist/actions/organize-imports/

I have turned it off for now.